cognite/_utils.py

```diff
@@ -123,6 +126,30 @@ def _get_first_datapoint(tag, start, end, api_key, project):
         return int(res[0]['timestamp'])
     return None
 
+def get_last_datapoint_ts(tag, api_key, project):
+    '''Returns the timestamp of the last datapoint of a timeseries.'''
+    from cognite.timeseries import get_latest
+    return int(get_latest(tag, api_key=api_key, project=project).to_json()['timestamp'])
+
+
+def interval_to_ms(start, end):
+    '''Returns the ms representation of start-end-interval whether it is time-ago, datetime or None.'''
+    time_now = int(round(time.time() * 1000))
+    if isinstance(start, datetime):
+        start = datetime_to_ms(start)
+    elif isinstance(start, str):
+        start = time_now - _time_ago_to_ms(start)
+    elif start is None:
+        start = time_now - _time_ago_to_ms('1w-ago')
+
+    if isinstance(end, datetime):
+        end = datetime_to_ms(end)
+    elif isinstance(end, str):
+        end = time_now - _time_ago_to_ms(end)
+    elif end is None:
+        end = time_now
+
```

Comment:
do we have default value for 'start' as well? Afaik, it is 1w-ago for the public API.